ui/src/infra-model/projektivelho/pv-file-list.tsx

```diff
@@ -207,6 +304,107 @@ const PVFileListRow = ({
             ) : (
                 <></>
             )}
+            {showConfirmAssignmentReject && (
```

Comment:
Nämä 3 tästä alkavaa dialogia näyttävät hyvin samoilta, mutta käytännössä noin joka toisella dialogin rivillä on jotain pientä parametrisoitavaa -> tein erillisinä dialogeina. Ajattelin että on lopulta selkeämpi näin